web/src/main/java/co/mobileaction/example/web/model/User.java

```diff
@@ -0,0 +1,15 @@
+package co.mobileaction.example.web.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "users_min")
```

Comment:
```suggestion
@Table(name = "users")
```

May be a better name.